pkgs/by-name/gl/glock/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule {
+  pname = "glock";
+  version = "0-unstable-2024-06-09";
+
+  src = fetchFromGitHub {
+    owner = "robfig";
+    repo = "glock";
+    rev = "0ac7e52a4c8a9a7039a72b3c6a10e8be59bc6599";
+    hash = "sha256-EDaLk83u1gRcvEjrfBrLZBQZ5unyD9LQA2TccOawXII=";
+  };
+
+  vendorHash = "sha256-v3lfb+CXbTxzObDpubufD3Q1h6IhULcC/6spA6StfGw=";
+  patches = [ ./modules.patch ];
```

Comment:
Again, @luftmensch-luftmensch, should we open a PR with the changes upstream and fetch the patch here instead?

Reply:
Done!